metis-enrichment/metis-enrichment-rest/src/test/java/eu/europeana/enrichment/rest/enrichment/EntityClientResolverITTest.java

```diff
@@ -0,0 +1,82 @@
+package eu.europeana.enrichment.rest.enrichment;
+
+import eu.europeana.enrichment.api.external.impl.EntityClientResolver;
+import eu.europeana.enrichment.api.external.model.EnrichmentBase;
+import eu.europeana.enrichment.api.internal.*;
+import eu.europeana.enrichment.utils.EntityType;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+// TODO - This is a Integration Test. Once we have measured performance of the new implementation,
+//  proper junit test will be added.
+@Disabled
+public class EntityClientResolverITTest {
```

Comment:
We don't do those kind of integration tests in metis.
At least not yet.
If you want this kind of test, it should be implemented with Wiremock, otherwise we'll leave it disabled for now.

Reply:
Yes, we can leave it as disabled or we can remove it as well.

Reply:
Okay.